clients/acme.js

```diff
@@ -33,4 +33,4 @@ function thanks(payload) {
     capsConnect.emit('recieved', payload);
 }
 
-module.exports = thanks;
\ No newline at end of file
+module.exports = thanks;
```

Comment:
This all looks really great, I would like to see tests added (right now they won't run because they are looking for caps.js)and I would update the title on your README from lab-11 to something like CAPs Message System